R/engine_multiverse.R

```diff
@@ -149,9 +144,8 @@ multiverse_default_block_exec <- function(.code, options, knit = FALSE) {
     # only output character vectors and conditions (warnings, etc) values (not plots or
     # source code) here, as everything else (e.g. graphics, messages) will have already
     # been output during evaluate()
-    outputs[sapply(outputs, function(x) is.character(x) || is_condition(x))]
+    Filter(function(x) is.character(x) || is_condition(x), outputs)
```

Comment:
When `outputs` is length 0 (i.e. if there's a parse error), this ensures that you don't get an error because `sapply()` returns a `list()`.